languages.js

```diff
@@ -30,6 +30,7 @@ var instructionsUk = require('./languages/translations/uk.json');
 var instructionsVi = require('./languages/translations/vi.json');
 var instructionsYo = require('./languages/translations/yo.json');
 var instructionsZhHans = require('./languages/translations/zh-Hans.json');
+var instructionsFaIr = require('./languages/translations/fa-IR.json');
```

Comment:
This localization should be under the language code `fa`, not `fa-IR`. We shouldn’t add a localization specific to Iran until we have one that covers the Persian language regardless of the country. Otherwise, clients that don’t specify the country or that specify a different country won’t be able to pick up this localization at all. There may eventually be a need to split out a separate `fa-IR` localization, in order to account for any differences in terminology that are common in Iran but not abroad. However, this would be a premature step for now.

I’m not sure who approved the request to add the [`fa-IR`](https://app.transifex.com/project-osrm/osrm-text-instructions/language/fa_IR/) localization on Transifex, but the translations there should be merged with the localization for [`fa`](https://app.transifex.com/project-osrm/osrm-text-instructions/language/fa/) that was started there. If you need access to that localization in order to complete it, please put in a request to join the translation team. Thanks!